state-chain/pallets/cf-swapping/src/tests.rs

```diff
@@ -1872,6 +1917,7 @@ fn swap_scheduled_event_witnessed(
 	deposit_amount: AssetAmount,
 	destination_asset: Asset,
 	broker_commission: AssetAmount,
+	execute_at: u64,
```

Comment:
not part of your changes by the name of this function is a little strange, it has nothing to do with witnessing 🤔 . I think just `assert_swap_scheduled_event` like we do for other things is clearer

Reply:
Renamed to `assert_swap_scheduled_event_emitted`.